.github/workflows/smoke-test-release-run-ubuntu.yml

```diff
@@ -0,0 +1,101 @@
+name: "Positron: Run smoke tests against latest release"
+
+on:
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  linux:
+    name: Tests on Linux
+    runs-on: ubuntu-latest-8x
```

Comment:
Did you ever try 4x? Just curious how it fares since we've been chatting about it....

Reply:
Oh, I forgot, but I can try it.